stable/mongodb/README.md

```diff
@@ -60,6 +60,7 @@ The following table lists the configurable parameters of the MongoDB chart and t
 | `image.debug`                                      | Specify if debug logs should be enabled                                                                                                                   | `false`                                                  |
 | `nameOverride`                                     | String to partially override mongodb.fullname template with a string (will prepend the release name)                                                      | `nil`                                                    |
 | `fullnameOverride`                                 | String to fully override mongodb.fullname template with a string                                                                                          | `nil`                                                    |
+| `rbacEnabled`                                      | Specify if rbac is enabled in your cluster                                                                                                                | `true`                                                  |
```

Comment:
see https://helm.sh/docs/topics/chart_best_practices/rbac/ for best practices

Reply:
Thanks for the heads up. Fixed it